### PR TITLE
#503/persist queued/playing UI status and action

### PR DESCRIPTION
**Pull request by Damrod**
As described in #503

### CODE REVIEW

File: webapp/src/shared/components/Button/Button.css [hunk 1]
```diff
@@ -65,3 +65,10 @@
   position: absolute;
   left: 0.675rem;
 }
+
+.button-bubble {
+  z-index: 99;
+  position: fixed;
+  right: -0.5rem;
+  top: -0.5rem;
+}
```
Comment (fizcamposancos) [inline webapp/src/shared/components/Button/Button.css:74]: Don´t you think the cropped button looks a bit weird ? What about something like this?

```suggestion
}.button-bubble {
  z-index: 99;
  position: fixed;
  right: -0.2rem;
  top: -0.2rem;
}

.button-bubble > .button-chip {
  border-radius: 0.5rem;
}
```
```

**Review by fizcamposancos — APPROVED** [review]
I like the idea of showing in the front end the waiting state, and making it interactive

**Comment by Damrod** [comment]
I'm going to be thinking a bit more about the button. Maybe create a new component. I'm not satisfied.

**Comment by Damrod** [comment]
I think now it's a bit prettier. Please, comment.

File: webapp/src/shared/components/Button/Button.css (hunk 1 above)
Comment (Damrod) [reply webapp/src/shared/components/Button/Button.css]: take a look at the latest proposition, please

File: webapp/src/shared/components/Button/CollapsibleButton.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  height: 3rem;
+  width: auto;
+  position: relative;
+}
+
+.collapsible-button-uncollapsed .button-chip {
+  z-index: 98;
+  width: 100%;
+  position: absolute;
+  top: 0.5rem;
```
[the remark this answers is not in the document]
Comment (Damrod) [reply webapp/src/shared/components/Button/CollapsibleButton.css]: done

**Pull request merged**